src/mrfgen/mrfgen.py

```diff
@@ -1127,6 +1267,34 @@ def create_vrt(basename, empty_tile, epsg, xmin, ymin, xmax, ymax):
             noaddo = True
     except:
         noaddo = False
+
+    # mrf_cores (max number of cpu cores to run on if mrf_parallel is set, defaults to 4
+    try:
+        mrf_cores = int(get_dom_tag_value(dom, 'mrf_cores'))
+    except:
+        mrf_cores = 4 # multiprocessing.cpu_count()
+
+    # mrf_parallel (run mrf_insert in parallel), defaults to False
+    try:
+        if get_dom_tag_value(dom, 'mrf_parallel') == "true":
+            mrf_parallel = True
+        else:
+            mrf_parallel = False
+    except:
+        mrf_parallel = False
+
+    # run the mrf_clean utility to reduce the size of the generated MRFs, defaults to False unless parallel is true
```

Comment:
Could also update the comment and assignment to be "defaults to `mrf_parallel`"

Reply:
Done.